.github/workflows/compatibility.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         pack_kind: [current, previous]
         create_builder_kind: [current, previous]
-        lifecycle_kind: [current, previous]
+        lifecycle_kind: [default, previous]
```

Comment:
I still sort of wonder about this change. Right now, `current` means "latest released"; `previous` means "latest patch of previous minor"; `default` means "whatever pack thinks is the default". It's possible these could all be different. I don't know what makes the most sense to automatically test. Also we should probably document these definitions somewhere (wherever we land).

Reply:
Responding in a [consistent location](https://github.com/buildpacks/pack/pull/708#discussion_r445808594)